packages/SystemsThinkingWorkbench-Core.package/STWSimulation.class/instance/stepSimulation.st

```diff
@@ -1,4 +1,9 @@
 simulation
 stepSimulation
+	"Decreases the startNode level by the specified output, increases the endNode level by the specified intake (for every connection) and steps up the simulationTime."
 
+	self connections
+		do: [:connection | 
```

Comment:
Please use `Connection>>performSimulationStep`, since the logic below will depend on the connection type